lib/ui/onboarding.dart

```diff
@@ -20,151 +22,260 @@ class OnboardingScreen extends StatefulWidget {
-              width: personaRenderWidth)),
+          left: availableWidth,
+          child: SizedBox(
+              width: 2 * availableWidth,
+              height: availableHeight,
+              child: SvgPicture.asset('assets/images/bodyLulo.svg'))),
       Positioned(
-          left: bodySpacing * 2 - leftOffset,
-          child: SvgPicture.asset('assets/images/bodyAntivacina.svg',
-              width: personaRenderWidth)),
+          left: 2 * availableWidth,
+          child: SizedBox(
+            width: 2 * availableWidth,
+            height: availableHeight,
+            child: SvgPicture.asset(
+              'assets/images/bodyAntivacina.svg',
+            ),
+          )),
       Positioned(
-          left: bodySpacing * 3 - leftOffset,
-          child: SvgPicture.asset('assets/images/bodyVegana.svg',
-              width: personaRenderWidth)),
+          left: 3 * availableWidth,
+          child: SizedBox(
+            width: 2 * availableWidth,
+            height: availableHeight,
+            child: SvgPicture.asset(
+              'assets/images/bodyVegana.svg',
+            ),
+          )),
       Positioned(
-          left: bodySpacing * 4 - leftOffset,
-          child: SvgPicture.asset('assets/images/bodyTump.svg',
-              width: personaRenderWidth)),
+          left: 4 * availableWidth,
+          child: SizedBox(
+            width: 2 * availableWidth,
+            height: availableHeight,
+            child: SvgPicture.asset(
+              'assets/images/bodyTump.svg',
+            ),
+          )),
       Positioned(
-          left: bodySpacing * 5 - leftOffset,
-          child: SvgPicture.asset('assets/images/bodyBonosaro.svg',
-              width: personaRenderWidth)),
+          left: 5 * availableWidth,
+          child: SizedBox(
+            width: 2 * availableWidth,
+            height: availableHeight,
+            child: SvgPicture.asset(
+              'assets/images/bodyBonosaro.svg',
+            ),
+          )),
       Positioned(
-          left: bodySpacing * 6 - leftOffset,
-          child: SvgPicture.asset('assets/images/bodyEronMust.svg',
-              width: personaRenderWidth)),
+          left: 6 * availableWidth,
+          child: SizedBox(
+            width: 2 * availableWidth,
+            height: availableHeight,
+            child: SvgPicture.asset(
+              'assets/images/bodyEronMust.svg',
+            ),
+          )),
       Positioned(
-          left: bodySpacing * 7 - leftOffset,
-          child: SvgPicture.asset('assets/images/bodyTiadoZap.svg',
-              width: personaRenderWidth)),
+          right: 0,
+          child: SizedBox(
+            width: 2 * availableWidth,
+            height: availableHeight,
+            child: SvgPicture.asset(
+              'assets/images/bodyTiadoZap.svg',
+            ),
+          )),
     ]);
   }
 
-  Widget pageBuilder(Text balloonHeader, RichText balloonText, Widget illustration) {
+  Widget pageBuilder(
```

Comment:
What is the reason for not using `pageBuilder` anymore?

Reply:
refactored cote to use pageBuilder